setenergy_lsdc.py

```diff
@@ -703,15 +759,19 @@ def setELsdc(energy,
         print('ivu_gap_scan() successful')
         time.sleep(1)
     
-    # Activate sector 17 photon local feedback
-    photon_local_feedback_c17.x_enable.put(1)
-    photon_local_feedback_c17.y_enable.put(1)
+    # Activate sector 17 photon local feedback (Legacy)
+    # photon_local_feedback_c17.x_enable.put(1)
+    # photon_local_feedback_c17.y_enable.put(1)
+
+    # Hold pitch
+    yield from activate_pitch_hold()
     
     # Align LSDC microscope center to beam center
     if beamCenterAlign:
         # Check for pre-conditions for beam_center_align()
         if shutter_hutch_c.status.get():
-            print('Experiment hutch shutter closed. Has to be open for this to work. Exiting')
+            print('Experiment hutch shutter closed. Has to be open for this to work. Stopping')
```

Comment:
better to refactor the message and use in print (or logging?) and gui_message

Reply:
Refactored in https://github.com/NSLS-II/lsdc/pull/369/commits/f060df6fe1bb7be3855ca46033fd6bef7c13808e